api/src/services/users.js

```diff
@@ -464,7 +464,7 @@ module.exports = {
   createUser: data => {
     const missing = missingFields(data);
     if (missing.length > 0) {
-      return Promise.reject(error400('Missing required fields: ' + missing.join(', ')));
+      return Promise.reject(error400({ message: 'Missing required fields: ' + missing.join(', '), translationKey: 'fields.required' }));
```

Comment:
We could add a translation parameter here to list the missing required fields in the translation as well.